tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -9,27 +10,31 @@
 def test_cb_trace_norm_quantum_channel():
     """The diamond norm of a quantum channel is 1."""
     phi = dephasing(2)
-    np.testing.assert_equal(completely_bounded_trace_norm(phi), 1)
+    assert completely_bounded_trace_norm(phi) == 1
+
+
+def test_cb_trace_norm_CP():
+    """Test for the diamond norm of a CP map."""
+    non_normalized_depolarizing_array = np.eye(4)
+    assert completely_bounded_trace_norm(non_normalized_depolarizing_array) == 4.0
 
 
 def test_cb_trace_norm_unitaries_channel():
-    """The diamond norm of phi = id- U id U* is the diameter of the smallest circle that contains the eigenvalues of U."""
+    """The diamond norm of phi = id- U id U* is the diameter of the smallest circle
```

Comment:
I wonder if the linter is over-aggressively making new lines here when it shouldn't be?

Reply:
Probably due to us having different configuration files with incompatible line lengths.